### PR TITLE
BIGC-725: allow product variants in cart

### PR DESCRIPTION
**Pull request by Menes1337**

### CODE REVIEW

File: extension/test/integration/extension/lib/bigcommerce/CartRepositoryTest.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const BigCommerceCartRepository = require('../../../../../extension/lib/bigcommerce/CartRepository')
-const BigCommerceFactory = require('../../../../../extension/lib/bigcommerce/Factory')
-const integrationCredentials = require('../../../../../.integration-credentials')
-const ShopgateCartFactory = require('../../../../../extension/lib/shopgate/CartFactory')
+const BigCommerceCartRepository = require('../../../../../lib/bigcommerce/CartRepository')
+const BigCommerceFactory = require('../../../../../lib/bigcommerce/Factory')
+const integrationCredentials = require('../../../../../../.integration-credentials')
```
Comment (chrisEff) [inline extension/test/integration/extension/lib/bigcommerce/CartRepositoryTest.js]: This does not work for me. The folder seemed to be correct before.

Comment (chrisEff) [reply extension/test/integration/extension/lib/bigcommerce/CartRepositoryTest.js]: fixed it

**Comment by Menes1337** [comment]
@chrisEff I just checked the issues and it looks like BigCommerce changed something at their API. I made some changes accordingly to our code base.

Please do a second code review

File: extension/test/integration/extension/lib/bigcommerce/CartRepositoryTest.js [hunk 2]
```diff
@@ -101,7 +106,13 @@ describe('BigCommerceCartRepository - integration', () => {
     shopgateCart.totals.should.containSubset([{
       _type: 'grandTotal',
       _label: 'Total',
-      _amount: 42.25,
+      /**
+       * BigCommerce currently returns $46.05 as cart_amount probably due to an API change between
+       * Fr 09th May 16:00 (MEZ) and Fr 09th May 16:00 (MEZ) and Mo 14th May 14:36 (MEZ)
```
Comment (chrisEff) [inline extension/test/integration/extension/lib/bigcommerce/CartRepositoryTest.js]: You mentioned "Fr 09th May 16:00 (MEZ)" twice :-)

**Pull request merged**